src/index.html

```diff
@@ -9,6 +9,11 @@
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-  <h1>Search bar airbnb</h1>
+  <form class="form form-big"  data-qa="big">
```

Comment:
давай модифікатори писати через нижнє підкреслення
```suggestion
  <form class="form form_big"  data-qa="big">
```

Reply:
я думаю краще назвати `search-form`

Reply:
назва класу не має бути назвою тега

Reply:
ok